src/locales/en.json

```diff
@@ -118,21 +118,34 @@
 	},
 	"pageSettings": {
 		"addPasskey": "Add passkey",
+		"deleteAccount": {
+			"buttonText": "Delete your account",
+			"deleteButtonTitleLocked":"You cannot delete your account while locked",
+			"description": "Once you delete your account, there is no going back. Please be certain.",
+			"messageDeletePasskeyPart1": "Are you sure you want to delete",
+			"messageDeletePasskeyPart2": "passkey?",
+			"messageDeleteAccount": "Are you sure you want to delete <strong> your Account </strong>?",
```

Comment:
These are all unused, instead the messages are hard-coded:

- [`<ConfirmDeletePopup>` to delete passkey](https://github.com/wwWallet/wallet-frontend/pull/167/files#diff-9760b7babfb1eae127ff9f01f25c6494cd24d2e638591ebfb83db7e8885fadb0R534)
- [`<ConfirmDeletePopup>` to delete account](https://github.com/wwWallet/wallet-frontend/pull/167/files#diff-9760b7babfb1eae127ff9f01f25c6494cd24d2e638591ebfb83db7e8885fadb0R742)